torch/_inductor/kernel/mm_scaled.py

```diff
@@ -207,10 +219,11 @@ def scaled_mm_options(  # type: ignore[no-untyped-def]
         sympy.gcd(sym_k, config.kwargs["BLOCK_K"]) == config.kwargs["BLOCK_K"]
     )
 
-    assert len(scale_a.get_size()) == len(
-        scale_b.get_size()
-    ), "Expect inverse scale_a and scale_b to be both scalars (tensor-wise scaling) or tensors (rowwise scaling)."
-
+    size_a, size_b = scale_a.get_size(), scale_b.get_size()
+    assert are_compatible_scales(size_a, size_b), (
+        f"Expect scale_a and scale_b to be either both scalars (including single-element tensors) "
```

Comment:
Probably shouldn't be an f-string, ruff will likely complain

Reply:
ahhh yeah will change